plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -72,10 +72,18 @@ contract PaymentInFlightExitRouter is IExitProcessor, Operated, OnlyWithValue {
             supportedTxType
         );
 
+        uint256 ethVaultId = framework.vaultToId(address(ethVault));
```

Comment:
I would replace injection of vault address -> vault id for `Router` too. And fetch vault contract here instead. It is more client friendly for them to inject with `id` instead of `address` which would differs on different deployment environment.

Reply:
That's a good idea!

Reply:
done